src/raspi/samples/pwm_led.py

```diff
@@ -0,0 +1,43 @@
+# Code for testing the Raspberry Pi's PWM signal capabilities.
+# Code from https://randomnerdtutorials.com/raspberry-pi-pwm-python/#:~:text=Python%20Script%20%E2%80%93%20PWM%20on%20Raspberry%20Pi%20GPIOs
+
+from gpiozero import PWMLED
+from time import sleep
+
+PINLIST = [14, 15, 18, 23, 4, 17, 27, 22]
```

Comment:
Not a huge deal but I do usually like to put some sort of documenting comment over big lists of numbers like this just to specify what these pins are